appveyor.yml

```diff
@@ -13,4 +13,4 @@ install:
 build_script:
   - go build ./agent
 test_script:
-  - for /f "" %%G in ('go list github.com/aws/amazon-ecs-agent/agent/... ^| find /i /v "/vendor/"') do ( go test -race -tags unit -timeout 40s %%G & IF ERRORLEVEL == 1 EXIT 1) 
+  - for /f "" %%G in ('go list github.com/aws/amazon-ecs-agent/agent/... ^| find /i /v "/vendor/"') do ( go test -race -tags unit -timeout 40s %%G & IF ERRORLEVEL == 1 EXIT 1)
```

Comment:
Should we increase the timeout to 60 here as well?

Reply:
Forgot to update summary after I decided to use a customized backoff to shorten runtime. I didn't go with the route to update longer unit test timeout. The summary should be updated now.